app/components/common/MapUK.component.js

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState, useCallback, useRef } from "react";
+import PropTypes from "prop-types";
+import {
+  Marker,
+  GoogleMap,
+  InfoWindow,
+  DirectionsRenderer,
+} from "@react-google-maps/api";
+import mapStyles from "./mapStyles";
+import { getInitials } from "./Avatar.component";
+
+const containerStyle = {
+  width: "100%",
+  height: "80vh",
+};
+
+const center = {
+  lat: 54.2511,
+  lng: -4.4632,
+};
+
+const options = {
+  styles: mapStyles,
+  disableDefaultUI: true,
+  zoomControl: true,
+};
+
+const MapUK = ({ teams }) => {
+  const London = new window.google.maps.LatLng(51.509865, -0.118092);
+  const Edinburgh = new window.google.maps.LatLng(55.953251, -3.188267);
+
+  const [selected, setSelected] = useState(null);
+  const [markers, setMarkers] = useState([]);
+  const [directions, setDirections] = useState([]);
```

Comment:
If you set this to null then you remove the "InvalidValueError: setDirections: in property routes: not an Array" errors

Reply:
Done!